xml/System.Runtime.Serialization/ImportOptions.xml

```diff
@@ -107,7 +107,7 @@
       <Docs>
         <summary>Gets or sets a data contract surrogate that can be used to modify the code generated during an import operation.</summary>
         <value>An implementation of the <see cref="T:System.Runtime.Serialization.IDataContractSurrogate" /> interface that handles schema import.</value>
-        <remarks>To be added.</remarks>
+        <remarks>The interface type for this option is ISerializationSurrogateProvider, but to take full advantage of the imported code modification abilities, using an <see cref="T:System.Runtime.Serialization.ISerializationSurrogateProvider2" />ISerializationSurrogateProvider2 that also implements <see cref="T:System.Runtime.Serialization.ISerializationCodeDomSurrogateProvider" /> is recommended.</remarks>
```

Comment:
The `ISerializationSurrogateProvider2` API will be available in RC2. The warning will go away after we update the mdoc ref assemblies to RC2.

https://github.com/dotnet/runtime/blob/release/7.0/src/libraries/System.Runtime.Serialization.Primitives/src/System/Runtime/Serialization/ISerializationSurrogateProvider2.cs